stdlib/src/mmap.rs

```diff
@@ -112,9 +112,28 @@ mod mmap {
     #[pyattr]
     use libc::{
         MADV_DODUMP, MADV_DOFORK, MADV_DONTDUMP, MADV_DONTFORK, MADV_HUGEPAGE, MADV_HWPOISON,
-        MADV_MERGEABLE, MADV_NOHUGEPAGE, MADV_REMOVE, MADV_SOFT_OFFLINE, MADV_UNMERGEABLE,
+        MADV_MERGEABLE, MADV_NOHUGEPAGE, MADV_REMOVE, MADV_UNMERGEABLE,
     };
 
+    #[allow(unused_imports)]
```

Comment:
This is supposed to be used by `#[pyattr]`

```suggestion
    #[pyattr]
```

Because we usually use cfg - pyattr order, placing it under cfg will be better.

Reply:
Fixed